code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java

```diff
@@ -30,18 +30,24 @@ String getServletURIName() {
   @Override
   void useCredential(String userId, Credential credential, HttpServletResponse response) throws IOException, InterruptedException {
     // Execute runnable to get people data
-    Set<PotentialContact> contactsFromCalendar = new HashSet<PotentialContact>();
+    CalendarDataForPositiveUser calendarDataForPositiveUser = new CalendarDataForPositiveUser(ofy(), userId, credential);
     Thread peopleInfo = new Thread(new PeopleDataForPositiveUser(ofy(), userId, credential));
-    Thread contactInfo = new Thread(new CalendarDataForPositiveUser(ofy(), userId, credential, contactsFromCalendar));
+    Thread contactInfo = new Thread(calendarDataForPositiveUser);
 
     peopleInfo.start();
     contactInfo.start();
     
     peopleInfo.join();
     contactInfo.join();
 
-    // TODO Load PositiveUserContacts from objectify
-    // TODO call mergeContactListsFromPeopleAPI(contactsFromPeople)
+    try {
+      PositiveUserContacts fullContacts = ofy.load().type(PositiveUserContacts.class).id(userId).now();
```

Comment:
lets not do a try catch, we want the application to proceed even if we dont use contacts api, just check that fullContacts is not null and if it is make a new object to load into objectify

Reply:
+1